api/serializers/statuses.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+
+from api.models.statuses import StudentStatus, StudentStatusName, TeacherStatus, TeacherStatusName
```

Comment:
these models will be gone, so your life will hopefully be easier :)
I hope that #10 gets merged into `master` real soon so you can merge `master` into your branch and get rid of `Status` and `StatusName`s

Reply:
UPD: PR got merged, I updated your branch by merging `master` into it.